rocket-cookie-auth/examples/example.rs

```diff
@@ -0,0 +1,74 @@
+use anyhow::Result;
```

Comment:
I don't understand the question. :)

Reply:
I was wondering if we could trim down the new crate any further.

For instance, I would just remove the example (since it's not meaningfully different from the original) and all the other files related to the example. It's not like we're gonna use this library anywhere else.